### PR TITLE
[Website] Highlights Endpoint Integration

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)
- Features integration of highlights endpoint to serve the highlights section data in frontend. Also includes new loadspinners for section loading and image loading.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
- git checkout staging
- git pull origin staging
- git checkout origin _ft-highlights-integration_ 
- To make changes, make sure to have your _postgres_ database set up and _URI_ updated in your local _.env_ file. Also update the following ENV variable: _REACT_APP_WEBSITE_BASE_URL=http://localhost:8000/api/v1_ 
- Finally, run the _serve_ script and _django server_  to add data through the admin portal and see it served in the frontend. [Instructions](https://github.com/airqo-platform/AirQo-frontend/edit/staging/website/README.md#running-the-website-application)

#### What are the relevant tickets?
- [WEB-295]

#### Screenshots
Admin Portal
Screenshot from 2022-08-10 20-41-19

Frontend
Screenshot from 2022-08-10 20-41-25

[WEB-295]: https://airqoteam.atlassian.net/browse/WEB-295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: netmanager/package-lock.json [hunk 1]
```diff
@@ -10450,7 +10450,7 @@
             "integrity": "sha512-J9X76xnncMw+wIqb15HeWfPMqPwYxSpPY8yWPJ7rAZN/ZDzFkjCSZObryCyUe8zbrVRNiuCnIeQteCzMn7GnWw==",
```
Comment (Baalmart) [inline netmanager/package-lock.json:10450]: This file change should not be part of this PR.

**Review by Baalmart — CHANGES_REQUESTED** [review]
Thanks @FaithDaka for this. Brilliant stuff!

1. Just requesting one small change as already highlighted in a review comment. Ideally, we should not be changing files from other products in this website PR. That is usually a red flag.
2. Also, could you add some "PR review comments" just to quickly onboard/guide the reviewers who are not actively contributing to the codebase? This eliminates the need for you to take us through the PR on video calls.
3. Lastly, we shall need to update the README to include the quick run instructions which were recently shared via Slack. The instruction in the README are quite verbose at the moment - which is also very good. In summary, the quick instructions for running which should be added are:
Ensure that you are running node v12, then:
```
cd website
npm install
npm run standalone (starts react project on localhost:8081)
```

Thanks again!

File: netmanager/package-lock.json (hunk 1 above)
Comment (FaithDaka) [reply netmanager/package-lock.json:10450]: Working on that

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
Outdated
